### PR TITLE
IGAPP-795: Separation lines in news lists

### PR DESCRIPTION
**Pull request by Cmd190**
Added separation lines in TuNews list. The same color as on native is used. See https://issues.tuerantuer.org/browse/IGAPP-795. 

Looks like this (textSecondaryColor):

image

### CODE REVIEW

File: native/src/components/NewsListItem.tsx [hunk 1]
```diff
@@ -41,9 +41,10 @@ const ListItemWrapper = styled.View`
 const StyledTouchableOpacity = styled.TouchableOpacity`
   flex-direction: column;
 `
+
 const Divider = styled.View<{ firstItem: boolean }>`
   border-top-width: 0.5px;
-  border-top-color: rgba(168, 168, 168, 0.7);
+  border-top-color: ${props => props.theme.colors.newsListSeparatorColor};
```
[the remark this answers is not in the document]
Comment (klinzo) [reply native/src/components/NewsListItem.tsx:47]: Please try to reuse an existing color like textSecondaryColor :) If necessary change on native as well.

[the remark this answers is not in the document]
Comment (Cmd190) [reply native/src/components/NewsListItem.tsx]: @klinzo  I altered it to text secondary color. Imo its looks fine.

**Review by klinzo — APPROVED** [review]
Looking good!

**Comment by Cmd190** [comment]
> pls use theme colors wherever possible

@f1sh1918 done

**Review by Taggotty — APPROVED** [review]

**Comment by Taggotty** [comment]
tested on chrome and android emulator.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**